pkg/sqlite/load.go

```diff
@@ -67,7 +67,7 @@ func (s *sqlLoader) AddOperatorBundle(bundle *registry.Bundle) error {
 }
 
 func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error {
-	addBundle, err := tx.Prepare("insert into operatorbundle(name, csv, bundle, bundlepath, version, skiprange, replaces, skips) values(?, ?, ?, ?, ?, ?, ?, ?)")
+	addBundle, err := tx.Prepare("insert into operatorbundle(name, csv, bundle, bundlepath, version, skiprange, replaces, skips, substitutesfor) values(?, ?, ?, ?, ?, ?, ?, ?, ?)")
```

Comment:
is there a reason why we shouldn't use `properties` table w/ a `substitutesFor` property and forgo the database schema changes?

Reply:
no reason - what is better in your opinion wrt declaritive config?

Reply:
this seems to be exactly the kind of the thing properties were made for.

Reply:
To clarify -- I think we should use properties here. I believe this PR can be simplified quite a bit.

I would like to hear some other voices though (cc @benluddy @ecordell @joelanford)

Reply:
My preference would be to reduce the number of properties that directly translate to upgrade edges. From a user perspective, if I see a declarative config that uses 3 or 4 different properties (channel, skips, skipRange, substitutesFor), it becomes harder and harder to build a mental model of that and figure out when I should use one property over another.

Reply:
See https://github.com/operator-framework/operator-registry/pull/617#discussion_r605121223

Reply:
we're banking on the visualization work being done but this is a good point. What alternative are you suggesting?